packages/@coorpacademy-components/src/molecule/card-with-button/style.css

```
@@ -0,0 +1,126 @@
+@value colors: "../../variables/colors.css";
+@value defaultText from "../../variables/fonts.css";
+
+.container { 
+    box-shadow: 0 0 4px 1px rgba(0, 0, 0, 0.1);
+    width: auto;
+    height: auto;
+    border-radius: 12px;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+.imageBox{
+    position: relative;
+    background-position: center center;
+	background-size: cover;
+	background-repeat: no-repeat;
+    width: 100%;
+    height: 100%;
+    border-radius: 12px 12px 0px 0px;
+    overflow: hidden; 
+}
+
+.content{
+    position: relative;
+    z-index: 1;
+}
+
+.zoom::after {
+    content: "";
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background: inherit;
+    background-size: cover;
+    transform-origin: center;
+    transition: all 1s ease-out;
+}
+
+.filtre{
+    width:100%
+}
+
+.filtre:after{
+    content:'\A';
+    position: absolute;
+    width:100%; height:100%;
+    top:0; left:0;
```

chaque propriété à la ligne
```suggestion
    top:0;
    left:0;
```